src/BSON/Binary.c

```diff
@@ -184,60 +184,6 @@ static PHP_METHOD(MongoDB_BSON_Binary, jsonSerialize)
 	ADD_ASSOC_STRINGL(return_value, "$type", type, type_len);
 }
 
-static PHP_METHOD(MongoDB_BSON_Binary, serialize)
-{
-	php_phongo_binary_t* intern;
-	zval                 retval;
-	php_serialize_data_t var_hash;
-	smart_str            buf = { 0 };
```

Comment:
You should be able to remove includes for `zend_smart_str.h` and `ext/standard/php_var.h` for all of these class files. Those were originally added for the Serializable implementations (e.g. https://github.com/mongodb/mongo-php-driver/commit/8ee86a15eb0c4d22ac41a7e8a0739d66594f4ce2).

Reply:
Ah, indeed 👍